salt/pillar/profiles.sls

```diff
@@ -20,6 +22,8 @@ profiles:
         port: 5432
     logging:
         level: DEBUG
+    orcid_dummy:
```

Comment:
this has been removed in favor of `pillar.elife.php_dummies.orcid_dummy`, should not be necessary